test/test_other.py

```diff
@@ -13492,6 +13505,26 @@ def test_unistd_close_noderawfs(self):
     self.emcc_args += ['--pre-js', 'pre.js']
     self.do_run_in_out_file_test('unistd/close.c')
 
+  @requires_node
+  def test_noderawfs_override_standard_streams(self):
```

Comment:
Should we call this test `test_module_stdout_stderr` and run it both with and without wasmfs?  (and move it up to be alongside `test_module_stdin`?

Reply:
Good idea, done with commit 02a4c50e247b4359a941e65d5108492b20b8afc7.